src/Tools/ExternalAccess/FSharp/Internal/VisualStudio/FSharpProjectExternalErrorReporterFactory.cs

```diff
@@ -19,7 +19,7 @@ internal static class FSharpProjectExternalErrorReporterFactory
         public static IVsLanguageServiceBuildErrorReporter2 Create(ProjectId projectId, string errorCodePrefix, IServiceProvider serviceProvider)
         {
             var workspace = (VisualStudioWorkspaceImpl)serviceProvider.GetMefService<VisualStudioWorkspace>();
-            return new ProjectExternalErrorReporter(projectId, errorCodePrefix, LanguageNames.FSharp, workspace);
+            return new ProjectExternalErrorReporter(projectId, projectId.Id, errorCodePrefix, LanguageNames.FSharp, workspace);
```

Comment:
is this the right guid?

Reply:
no, but it isn't super impactful.  However I think I can remove this, I'm not seeing any usages anywhere
https://github.com/search?q=org%3Adotnet%20FSharpProjectExternalErrorReporterFactory&type=code